tests/dummy/app/pods/demo/template.hbs

```diff
@@ -0,0 +1,33 @@
+<div class="demo-application">
+  <div class="demo-content">
+    {{frost-notifier}}
+    <form>
+        <fieldset>
+          <legend>Frost Notifications</legend>
+          <div class='field'>
+            <label>Message:</label>
+            <div class='field-value'>
+        		{{frost-text id="message-field" type="text" value=message on-input=(action 'text')}}
+            </div>
+          </div>
+          <div class='field'>
+            <label>Type:</label>
+            <div class='field-value'>
+          		{{frost-select data=selectData on-change=(action 'select')}}
+            </div>
+          </div>
+          <div class='field'>
+            <label>Timeout:</label>
+            <div class='field-value'>
+          		{{frost-text value=clearDuration}}
+            </div>
+          </div>
+          <div class='field'>
+            <label></label>
+      			{{#frost-button class="secondary" on-click=(action 'clearAll')}}Clear all{{/frost-button}}
```

Comment:
1. Indenting
2. It looks like this is using the old `frost-button` API instead of the `ember-frost-button` one (i.e. directly setting `class` instead of using `priority`.